src/main/java/address/controller/PersonCardController.java

```diff
@@ -45,13 +45,43 @@ public PersonCardController(Person person) {
 
     @FXML
     public void initialize() {
-        firstName.setText(person.getFirstName());
-        lastName.setText(person.getLastName());
-        street.setText(person.getStreet());
-        postalCode.setText(person.getPostalCodeString());
-        city.setText(person.getCity());
-        birthday.setText(person.getBirthdayString());
-        contactGroups.setText(getContactGroupsString(person.getContactGroupsCopy()));
+        firstName.textProperty().bind(person.firstNameProperty());
+        lastName.textProperty().bind(person.lastNameProperty());
+        street.textProperty().bind(person.streetProperty());
+        postalCode.textProperty().bind(new StringBinding(){
+            {
+                bind(person.postalCodeProperty()); //Bind property at instance initializer
+            }
+
+            @Override
+            protected String computeValue() {
+                if(person.postalCodeProperty().get() == 0){
+                    return "";
+                }
+                return String.valueOf(person.postalCodeProperty().get());
```

Comment:
What's the difference between this approach and using `postalCode.textProperty().bind(person.postalCodeProperty().asString()`?